### PR TITLE
Configuration update for CVE-2021-31920

### PR DESCRIPTION
**Pull request by mservidio**
[1] https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-31920
[2] https://cloud.google.com/anthos/run/docs/securing/best-practices

### All Submissions:

* [X] Have you followed the guidelines in our Contributing document?
* [X] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?

### CODE REVIEW

File: api/v1alpha/istio-manifests/1.4/authz/allow-nothing-default-policy.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+apiVersion: security.istio.io/v1beta1
+kind: AuthorizationPolicy
+metadata:
+  name: allow-nothing-default
+  namespace: default
+spec:
+  {}
```
Comment (mservidio) [inline api/v1alpha/istio-manifests/1.4/authz/allow-nothing-default-policy.yaml:8]: @phriscage I don't believe this one is needed as the sidecar injection is only enabled for datashare-apis from what I follow in the api docs:

https://github.com/GoogleCloudPlatform/datashare-toolkit/tree/master/api

Comment (phriscage) [reply api/v1alpha/istio-manifests/1.4/authz/allow-nothing-default-policy.yaml:8]: agreed. Feel free to remove

**Comment by mservidio** [comment]
@phriscage please review

**Review by phriscage — APPROVED** [review]
lgtm

**Pull request merged**